modules/appeals_api/app/models/appeals_api/veteran.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class Veteran
+    delegate_missing_to :@veteran
+
+    def initialize(ssn:, first_name:, last_name:, birth_date:)
+      @veteran = ClaimsApi::Veteran.new(
```

Comment:
Curious to hear your perspective on the advantage of wrapping the `ClaimsApi::Veteran` object, if we're passing any and all messages onto the wrapped object. What are we gaining over just instantiating a regular `ClaimsApi::Veteran` object?

Reply:
If we control the interface, we can make changes as needed to our public API, as well as switch out the internal implementation when/if we decide to talk to MPI ourselves. This decouples our client code (such as higher_level_review.rb lookup of ICN) from the claims implementation of Veteran.